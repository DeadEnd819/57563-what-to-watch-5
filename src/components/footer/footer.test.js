import React from "react";
import renderer from "react-test-renderer";
import Footer from "./footer";

jest.mock(`../logo/logo.jsx`, () => `Logo`);

it(`Should Footer render correctly`, () => {
  const tree = renderer
      .create(
          <Footer />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
