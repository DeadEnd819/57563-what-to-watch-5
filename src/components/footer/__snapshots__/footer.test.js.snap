// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer render correctly 1`] = `
<footer
  className="page-footer"
>
  <Logo
    className="logo__link--light"
  />
  <div
    className="copyright"
  >
    <p>
      Â© 2019 What to watch Ltd.
    </p>
  </div>
</footer>
`;
